gslib/commands/mb.py

```diff
@@ -154,6 +157,11 @@
   --retention time       Specifies the retention policy. Default is no retention
                          policy. This can only be set on gs:// buckets and
                          requires using the JSON API.
+
+  --pap setting          Specifies the public access prevention setting.
+                         Valid values are <enforced|unspecified>.
```

Comment:
Can we spell it out here? `Valid values are "enforced" or "unspecified".`

Also, should we be including these options in quotes, or in code format instead?

Reply:
fixed